ui/CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+<h2>Table of contents</h2>
+
+- [General contributor notes](#general-contributor-notes)
+  - [`feast ui` command](#feast-ui-command)
+  - [NPM package project structure](#npm-package-project-structure)
+  - [Tests](#tests)
+  - [Yarn commands](#yarn-commands)
+    - [`yarn install`](#yarn-install)
+    - [`yarn start`](#yarn-start)
+    - [`yarn test`](#yarn-test)
+- [Release process](#release-process)
+  - [(Advanced) Manually publishing the Feast Package to NPM](#advanced-manually-publishing-the-feast-package-to-npm)
+    - [Requirements](#requirements)
+    - [Steps for Publishing](#steps-for-publishing)
+
+# General contributor notes
+In this doc, we describe how to contribute both to the Feast Web UI NPM package as well as the embedded Feast UI in the Python SDK (i.e. what's run when you run `feast ui`)
+
+## `feast ui` command
+You can see the logic in [../sdk/python/ui](../sdk/python/feast/ui/). This instance is loaded in [../sdk/python/ui_server.py](../sdk/python/feast/ui_server.py). 
```

Comment:
I think you mean `../sdk/python/feast/ui` and `../sdk/python/feast/ui_server.py`

Reply:
done